src/libcrun/container.c

```diff
@@ -359,6 +359,26 @@ static const char *spec_cgroupns = "\
 				\"type\": \"cgroup\"\n\
 			},\n";
 
+static char *potentially_unsafe_annotations[] = {
+  "run.oci.handler",
+  "module.wasm.image/variant",
+  "run.oci.handler",
+  "run.oci.delegate-cgroup",
+  "io.kubernetes.cri.container-type",
+  "run.oci.handler",
+  "run.oci.systemd.subgroup",
+  "run.oci.mount_context_type",
+  "run.oci.systemd.force_cgroup_v1",
+  "run.oci.keep_original_groups",
+  "run.oci.pidfd_receiver",
+  "run.oci.hooks.stdout",
+  "run.oci.hooks.stderr",
+  "run.oci.seccomp.plugins",
+  "run.oci.seccomp.receiver",
+  "run.oci.seccomp_bpf_data",
+  "run.oci.seccomp_fail_unknown_syscall",
```

Comment:
is `run.oci.*` a valid value?

If so, that would be better, as potentially any `run.oci.*` annotation can change the behavior of the runtime

Reply:
not with the asterisk, but yeah I can do a generic one